src/mumble/PulseAudio.cpp

```diff
@@ -775,7 +775,7 @@ void PulseAudioSystem::stream_restore_read_callback(pa_context *c, const pa_ext_
 	}
 }
 
-void PulseAudioSystem::restore_volume_success_callback(pa_context *c, int success, void *userdata) {
+void PulseAudioSystem::restore_volume_success_callback(pa_context * /*c*/, int /*success*/, void *userdata) {
```

Comment:
Remove them then? I usually prefer commenting unused params so I don't have to look them up if I end up using them after all or just want to know what other params are available on the callback.